spec/classes/foreman_plugin_remote_execution_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+describe 'foreman::plugin::remote_execution' do
+  let :facts do
+    on_supported_os['redhat-7-x86_64']
```

Comment:
That seems like overkill, `foreman::plugin` is already well spec'd.  Salt was accepted before without one.  Should I add one for it too?

You'll be basically running the same spec a bunch of times.